core/commons/che-core-commons-observability/src/main/java/org/eclipse/che/commons/observability/LogWatcherMetricsImpl.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.commons.observability;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
+import javax.inject.Inject;
+
+/**
+ * Handles metrics of watching logs of the workspaces by binding it to given {@link MeterRegistry}.
+ */
+public class LogWatcherMetricsImpl implements LogWatcherMetrics {
+
+  private final Counter workspaceStartDebugMode;
+
+  @Inject
+  public LogWatcherMetricsImpl(MeterRegistry registry) {
+    workspaceStartDebugMode =
+        Counter.builder("workspace.start.debug.mode")
```

Comment:
Consider adding this counter as a tagged https://github.com/eclipse/che/blob/7a1d3964a6f3a58bbfbc9e98ffcbed0488bc403c/wsmaster/che-core-api-metrics/src/main/java/org/eclipse/che/api/metrics/WorkspaceStartAttemptsMeterBinder.java#L40. This is the Prometheus' way of "variants" of the same metric.

Reply:
done